frontend/lib/identification/qr_code_scanner/qr_code_scanner_page.dart

```diff
@@ -1,5 +1,5 @@
-import 'package:ehrenamtskarte/qr_code_scanner/qr_code_processor.dart';
-import 'package:ehrenamtskarte/qr_code_scanner/qr_code_scanner.dart';
+import 'package:ehrenamtskarte/identification/qr_code_scanner/qr_code_processor.dart';
+import 'package:ehrenamtskarte/identification/qr_code_scanner/qr_code_scanner.dart';
```

Comment:
Opening discussion here: I think we should just use "verification" here to get rid of "verification"

Reply:
[I opened an issue for this discussion.](https://github.com/digitalfabrik/entitlementcard/issues/721)